app/assets/javascripts/welcome.js

```diff
@@ -0,0 +1,13 @@
+// Loads all Bootstrap javascripts
+//= require jquery
+//= require jquery_ujs
+//= require bootstrap
+//= require footer_fixed.js
+//= require browser_timezone_rails/application.js
+
+$("a.github_auth").click(function () {
+  ga('send', 'event', 'button', 'click', {'page': '/users/auth/github'})
+  ga('send', 'event', 'button', 'click', {'page': '/users/auth/github'})
```

Comment:
__[staging][eslint]__
*  Error - "ga" is not defined. (no-undef)

*  Error - Strings must use doublequote. (quotes)

*  Error - Strings must use doublequote. (quotes)

*  Error - Strings must use doublequote. (quotes)

*  Error - Strings must use doublequote. (quotes)

*  Error - Strings must use doublequote. (quotes)

*  Error - Strings must use doublequote. (quotes)

*  Error - Missing semicolon. (semi)